AutomatticTracks/src/main/java/com/automattic/android/tracks/datasets/EventTable.java

```diff
@@ -97,11 +97,13 @@ public static void insertEvent(Context ctx, Event event) {
             }
 
             stmt.bindLong(8, event.getTimeStamp());
-            stmt.bindLong(9,  event.getRetryCount());
+            stmt.bindLong(9, event.getRetryCount());
 
             stmt.execute();
 
             db.setTransactionSuccessful();
+        } catch(IllegalArgumentException e) {
```

Comment:
I think we should add our checkstyle config in this project (there is a missing space between `catch` and `(`)